autoload/slime.vim

```diff
@@ -34,7 +34,8 @@ endfunction
 
 function! s:ScreenConfig() abort
   if !exists("b:slime_config")
-    let b:slime_config = {"sessionname": "", "windowname": "0"}
+    " STY is the 'Alternative socket name' which is set inside GNU Screen
+    let b:slime_config = {"sessionname": "" . $STY, "windowname": "0"}
```

Comment:
I am unclear if `"" . $STY` is preferred for handling a possibly unset $STY or if just $STY is sufficient and will return an empty string.  I tested both, but I'll admit that I didn't hammer on the tab completion aspect.  Using `"" . $STY` felt like it'd always give a string-valued result.  Suggestions much appreciated.

Reply:
I could also see this instead as a change to the "If you want vim-slime to prefill the prompt answers..." portion of the README.  Please shout if you'd prefer that.